docs/source/install.md

```diff
@@ -63,7 +63,7 @@ If you want to use a key-value store to mediate configuration
 (mainly for use in distributed deployments, such as containers),
 you can get the key-value stores via their respective installation pages:
 
-- Install [`redis-server`](https://redis.io/docs/install/install-redis/)
+- Install [`redis`](https://redis.io/docs/install/install-redis/)
```

Comment:
To be precise, this should be `redis-server`, e.g. `apt install redis-server` is correct, while `apt install redis` installs unnecessary things like the client.

Reply:
actually, it's just a metapackage that does the same thing 🤷 . No change to make!

Reply:
Ah, installing the `redis` package now via [3c5d754](https://github.com/jupyterhub/traefik-proxy/pull/231/commits/3c5d754da7bdc4c5037688efcc8dad577cb16af0), because that matches instructions on redis website as well